cmd/utils/cmd.go

```diff
@@ -40,7 +41,9 @@ import (
 )
 
 const (
-	importBatchSize = 2500
+	importBatchSize       = 2500
+	freeDiskSpaceWarning  = 1024 * 1024 * 1024
+	freeDiskSpaceCritical = 300 * 1024 * 1024
```

Comment:
Actually, this might be not a one-size-fits-all situation. I think these defaults are ok for mainnet, but if someone is running a 
- a temporary clique network, or
- a light node (on whatever network)

Then we shouldn't enforce this. 
So maybe only spin up the monitor if it's actually on mainnet and not in light mode?

Reply:
We could fix that by passing the chain id to StartNode
```
	utils.StartNode(stack,backend.ChainConfig().ChainID)
```

Reply:
That's fixed. I've also added a check for fast sync mode.